src/Microsoft.AspNetCore.Mvc.Core/Internal/DefaultModelBindingContext.cs

```diff
@@ -182,6 +185,8 @@ public override ModelBindingResult Result
             }
         }
 
+        private ILoggerFactory LoggerFactory { get; set; }
```

Comment:
`public virtual` (with doc comments) in `ModelBindingContext` and `override` here or just a field. Either way, should be get-only or `readonly`.